bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/form/InputBehavior.java

```diff
@@ -1,12 +1,18 @@
 package de.agilecoders.wicket.core.markup.html.bootstrap.form;
 
+import com.google.common.base.Function;
```

Comment:
Some people complained of the usage of Guava classes. Can we use java.util.function.Function instead ?

Reply:
I'd also prefer to avoid the Guava classes. I wasn't sure, if there is a deeper reason behind using the library and just didn't want to produce too many deviations from the existing code. I'm going to update the PR soon.